src/components/VideoPlayerContexts/VideoPopoverMenuContext.js

```diff
@@ -0,0 +1,47 @@
+import React, {useCallback, useMemo} from 'react';
+import PropTypes from 'prop-types';
+
+const VideoPopoverMenuContext = React.createContext(null);
+
+function VideoPopoverMenuContextProvider({children}) {
+    const [isPopoverVisible, setIsPopoverVisible] = React.useState(false);
+    const [anchorPosition, setAnchorPosition] = React.useState({vertical: 0, horizontal: 0});
```

Comment:
Let's be consistent and let's import `useState` same as `useCallback` and `useMemo`.